util/config.py

```diff
@@ -46,18 +46,14 @@ def load_config(self):
         """
         # Open the YAML config file and load its contents
         try:
```

Comment:
double nested try/catch was unnecessary. The outer catch is duplicated on line 52 (old), or line 51 (new)